container-admin/Dockerfile_client

```diff
@@ -43,16 +43,17 @@ COPY config /home/ubuntu/.consonance/config
 COPY key.pem /home/ubuntu/.ssh/key.pem
 COPY init_client.sh /home/ubuntu/init_client.sh
 
-RUN chmod 600 /home/ubuntu/.ssh/key.pem 
+RUN chmod 600 /home/ubuntu/.ssh/key.pem
 RUN chmod a+wrx /home/ubuntu/init_client.sh
 RUN chown -R ubuntu /home/ubuntu
 USER ubuntu
 RUN echo "postgres:5432:*:*:postgres" > /home/ubuntu/.pgpass && chmod 600 /home/ubuntu/.pgpass
 
 
-# for youxia and the consonance command line on the main box 
+# for youxia and the consonance command line on the main box
 RUN wget --no-verbose https://github.com/Consonance/consonance/releases/download/${consonance_version}/consonance && mv consonance /home/ubuntu/bin && chmod a+x /home/ubuntu/bin/consonance
```

Comment:
These binaries can just be attached to the release https://github.com/Consonance/consonance/releases if trying to avoid seqwaremaven.oicr.on.ca